templates/aerospike-cluster-entrypoint-new-vpc.template.yaml

```diff
@@ -115,42 +103,6 @@ Parameters:
     AllowedValues: ["2", "3"]
     Default: "3"
     Description: Number of Availability Zones to use in the VPC. This number must match the number of Availability Zones you chose for the AvailabilityZones parameter.
-  BastionInstanceType:
-    Default: t3.micro
-    Type: String
-    Description:  Type of EC2 instance for the bastion hosts.
-    AllowedValues:
-      - t2.nano
-      - t2.micro
-      - t2.small
-      - t2.medium
-      - t2.large
-      - t3.micro
-      - t3.small
-      - t3.medium
-      - t3.large
-      - m3.large
-      - m3.xlarge
-      - m3.2xlarge
-      - m4.large
-      - m4.xlarge
-      - m4.2xlarge
-      - m4.4xlarge
-  NumBastionHosts:
-    AllowedValues:
-      - '1'
-      - '2'
-      - '3'
-      - '4'
-    Default: '1'
-    Description: Number of bastion hosts to create.
-    Type: String
```

Comment:
We probably want to still take these but then pass them into the child stack.

Reply:
done